UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.API.FunctionalTests/Helper/ExchangeSetApiClient.cs

```diff
@@ -102,6 +105,22 @@ public async Task<HttpResponseMessage> GetProductIdentifiersDataAsync(List<strin
                 return await httpClient.SendAsync(httpRequestMessage, CancellationToken.None);
             }
         }
+
+        public async Task<HttpResponseMessage> PostEssWebhookAsync([FromBody] JObject request, string accessToken = null)
```

Comment:
As per rename in controller:
```suggestion
        public async Task<HttpResponseMessage> PostNewFilesPublishedAsync([FromBody] JObject request, string accessToken = null)
```